examples/commerce-essentials/src/app/(checkout)/checkout/DeliveryForm.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+import { CheckoutForm as CheckoutFormSchemaType } from "../../../components/checkout/form-schema/checkout-form-schema";
+import {
+  RadioGroup,
+  RadioGroupItem,
+} from "../../../components/radio-group/RadioGroup";
+import { Label } from "../../../components/label/Label";
+import { cn } from "../../../lib/cn";
+import {
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "../../../components/form/Form";
+import { useFormContext } from "react-hook-form";
+import { useShippingMethod } from "./useShippingMethod";
+import { LightBulbIcon } from "@heroicons/react/24/outline";
+import {
+  Alert,
+  AlertDescription,
+  AlertTitle,
+} from "../../../components/alert/Alert";
+import { Skeleton } from "../../../components/skeleton/Skeleton";
+
+
+import { useCart } from "@elasticpath/react-shopper-hooks";
+
+
+export function DeliveryForm() {
+  const { control } = useFormContext<CheckoutFormSchemaType>();
+  
+  const { data } = useCart();
+  const state = data?.state;
+  const { data: deliveryOptions } = useShippingMethod(state?.id);
+  
+  return (
+    <fieldset className="flex flex-col gap-6 self-stretch">
+      <div>
+        <legend className="text-2xl font-medium">Delivery</legend>
+      </div>
+      <Alert>
```

Comment:
Should we remove this alert given that we have plugged in a "shipping service"?

Reply:
Alert Removed